src/components/layout/VaultMenu.vue

```diff
@@ -17,7 +17,8 @@
         @click="() => loadMap(item.id)"
       >
         {{ item.label }}
-        <Checkmark class="menu-checkmark" v-if="item.active" />
+        <Checkmark class="menu-icon" v-if="item.active" />
+        <Close  class="menu-icon menu-delete" @click.stop="()=>deleteMap(item.id)" v-else/>
```

Comment:
Can you move this out to a separate button instead of just having a click handler on the icon? As it is it's not keyboard navigable, and I think we'll probably be adding additional actions in the future.

I'm thinking something like this, moving the button off of the `MenuItem` and into a separate button (and moving the styles off of `.menu-item` and onto `.menu-button`) and them move the close icon to a separate button as well.

```
  <MenuItem v-for="item in items" :key="item.id" as="div" class="menu-item">
    <button
      class="menu-button"
      :class="{ 'is-active': item.active }"
      :disabled="item.active"
      @click="() => loadMap(item.id)"
    >
      {{ item.label }}
      <Checkmark class="menu-icon" v-if="item.active" />
    </button>
    <button v-if="!item.active" @click.stop="() => deleteMap(item.id)" aria-label="Remove vault">
      <Close class="menu-icon menu-delete" />
    </button>
  </MenuItem>
```

Reply:
You are right, I will get these done as soon as possible.